cluster-autoscaler/cloudprovider/aws/auto_scaling_groups.go

```diff
@@ -315,49 +315,14 @@ func (m *asgCache) isPlaceholderInstance(instance *AwsInstanceRef) bool {
 	return strings.HasPrefix(instance.Name, placeholderInstanceNamePrefix)
 }
 
-// Fetch automatically discovered ASGs. These ASGs should be unregistered if
-// they no longer exist in AWS.
-func (m *asgCache) fetchAutoAsgNames() ([]string, error) {
-	groupNames := make([]string, 0)
-
-	for _, spec := range m.asgAutoDiscoverySpecs {
-		names, err := m.awsService.getAutoscalingGroupNamesByTags(spec.Tags)
-		if err != nil {
-			return nil, fmt.Errorf("cannot autodiscover ASGs: %s", err)
-		}
-
-		groupNames = append(groupNames, names...)
-	}
-
-	return groupNames, nil
-}
-
-func (m *asgCache) buildAsgNames() ([]string, error) {
-	// Collect explicitly specified names
-	refreshNames := make([]string, len(m.explicitlyConfigured))
+func (m *asgCache) getConfiguredAsgNames() ([]string, error) {
```

Comment:
you can probably remove the `error` from this internal signature since it can't really fail.

Reply:
Will remove